.circleci/config.yml

```diff
@@ -47,19 +48,32 @@ save-assets-cache: &save-assets-cache
     key: *assets-cache-key
     paths:
       - public/assets
-      - public/packs-test
       - tmp/cache/assets
-      - tmp/cache/webpacker
```

Comment:
I think we needed this in the past to make webpacker faster to identify that everything was present. Do you think it is not needed anymore? Or possibly was never needed?

Reply:
No idea to be honest 🤷🏻‍♂️